docs/lago-tutorial/README.md

```diff
@@ -0,0 +1,206 @@
+# lago-tutorial
+This repository contains the default files for creating Lago environment. 
+
+In this tutorial you will learn how to set up a custom virtual environment with Lago.
```

Comment:
This is not precise either, the idea is to have multiple scenarios, probably comment that here is the initial setup/prerequisites (installing lago, etc) and that each scenario will get the user through a different set of features/use cases where lago can help